bullet-train.zsh-theme

```diff
@@ -13,6 +13,9 @@
 # The default configuration, that can be overwrite in your .zshrc file
 # ------------------------------------------------------------------------------
 
+# space variable to make spaces more visible and avoid bugs
+space=' '
```

Comment:
This space is in quite a few places, but most of them are  `ZSH_THEME_GIT_PROMPT_` which is probably used later in 
https://github.com/robbyrussell/oh-my-zsh/blob/3705d47bb3f3229234cba992320eadc97a221caf/lib/git.zsh#L2